### DISCUSSION TITLE
running Plover 4 on Big Sur (developer cannot be verified)

### DISCUSSION THREAD

**Discussion by phques** [opensteno/plover#1235] [Q&A and Support] [answered]
Hi,
How do we get a current version of Plover to run on MacOS Big Sur?
I get an error about "developer cannot be verified" when I launch it.
thank you

**Comment by benoit-pierre** [comment]
We currently don't sign our application, so I think you'll have to bypass the security check: https://support.apple.com/guide/mac-help/open-a-mac-app-from-an-unidentified-developer-mh40616/mac.

**Reply by phques** [reply]
great, thank you !

**Comment by panathea** [comment, marked as answer]
Hi there. This happens because Plover is not code-signed. It's something that we're looking at accomplishing in the future, but there's a lot of work and an annual fee involved. So until then, I hope this helps!

1. Download the `.dmg` of Plover that you'd like to install.
2. Double-click to open the DMG.
3. A window will pop up with Plover and the applications folder, drag Plover into the Applications folder and wait for it to copy over.
    ![Dragging the Plover icon into the applications folder](https://user-images.githubusercontent.com/5840970/113603136-88e12900-9611-11eb-949b-df5969fc619e.png)
4. Launch Plover from the Applications folder. A warning will appear like the following:
    <img width="372" alt="Developer cannot be verified warning" src="https://user-images.githubusercontent.com/5840970/113603224-ad3d0580-9611-11eb-8552-a49876d16a73.png">
5. Press **Cancel** to dismiss the warning.
6. Open "System Preferences"
    
    ![image](https://user-images.githubusercontent.com/5840970/113603329-cf368800-9611-11eb-884f-630fbc58b5ef.png)
7. Go to "Security & Privacy"
    ![image](https://user-images.githubusercontent.com/5840970/113603574-1c1a5e80-9612-11eb-9533-4ab0e948a923.png)
8. In the **General** tab, you'll see a warning about Plover being from an unverified developer. You can choose to bypass this warning by clicking **Open Anyway**
    <img width="666" alt="The open anyway button in the security and privacy general section" src="https://user-images.githubusercontent.com/5840970/113603793-613e9080-9612-11eb-9915-14f7fe508f6c.png">
9. In the new, final warning that appears, click **Open** again.
<img width="372" alt="Final warning" src="https://user-images.githubusercontent.com/5840970/113603935-94811f80-9612-11eb-899c-cce3cad1829e.png">
10. Plover should be able to start now. If it doesn't start immediately, just launch it from Applications once more.

**Reply by phques** [reply]
great, thank you !

**Reply by panathea** [reply]
And as Benoit Pierre's linked article points out, you can get to the warning in step 9 by just holding down the `control` key while clicking Plover in the Applications folder!

**Reply by arxanas** [reply]
A recent version of macOS changed "System Preferences" to "System Settings" and now the UI looks different. You can still mostly follow the instructions, but Plover would appear somewhere around here now:

![Screenshot 2025-01-15 at 10 49 00 AM](https://github.com/user-attachments/assets/7b0c59f5-cbe2-4c51-9223-99bf06fd9df0)

**Comment by gdwaynewarner** [comment]
Any word on running Plover on Mac OS Sequioia (15.5)? Right now, I don't get any output from Plover.

**Reply by mkrnr** [reply]
Do you get any output in the paper tape? If yes, did you reset the accessibility permissions, see [this wiki page](https://plover.wiki/index.php/Troubleshooting_issues#macOS)?

Sadly, I can't test Plover on that macOS version.
